### DISCUSSION TITLE
Copilot doesn't active after following subscription steps

### DISCUSSION THREAD

**Discussion by mhazim2** [community/community#47927] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I follow the step to subscribe copilot and it was smooth until at the end of the step. But when i check back to the copilot section, it's still say subscription has ended and my card was not billed.

Steps:

1. Go to billing section and click `Enable Github Copilot`
![Screen Shot 2023-02-19 at 09 59 18](https://user-images.githubusercontent.com/15484312/219909931-08fac574-6b90-4a76-9e58-bdd750def834.png)

2. Select plan and click `Get access to Github Copilot`
![Screen Shot 2023-02-19 at 09 59 41](https://user-images.githubusercontent.com/15484312/219909952-4cd8a021-9054-499f-8745-878d5b669bc7.png)

3. Filling up payment details, payment method, and then click `Submit`
![Screen Shot 2023-02-19 at 10 00 09](https://user-images.githubusercontent.com/15484312/219910000-24516525-db3d-468e-89d4-77b44c1546a2.png)

4. Save my preference
![Screen Shot 2023-02-19 at 10 02 11](https://user-images.githubusercontent.com/15484312/219910089-fcf924eb-bf1f-4dd3-9742-b5e1504e91ad.png)

5. They said `Github Copilot is now ready to pair`
![Screen Shot 2023-02-19 at 10 02 28](https://user-images.githubusercontent.com/15484312/219910122-c9e72ff2-040c-432c-93c5-903e44840527.png)

6. But when i check back and refresh to the billing section it still ask for `Enabled Github Copilot`
![Screen Shot 2023-02-19 at 10 05 34](https://user-images.githubusercontent.com/15484312/219910222-bb45d9d2-9512-41e5-923f-f60c80f34daa.png)

Can someone advise me? My card is still active and i can use Copilot without any problem previously

**Comment by mhazim2** [comment]
For more context, previously when my subscription will ends, my balance was not enough. So Copilot automatically deactivated, but when i resolve balance issue and trying re-subscribe with the steps above it's works. So why now i can't do that? i was try to switch with other card and seems doesn't work.

**Comment by jhrldev** [comment]
I'm experiencing the same. After resolving the issue my card was charged but my subscription still not counting. Funny enough, I had one active sponsorship until 04/2023, but it's not there as well.

**Charged for the plan**
![image](https://user-images.githubusercontent.com/5360060/219932637-2e208643-2c97-40ba-a279-a165e2a9ed58.png)
**Still not activated**
![image](https://user-images.githubusercontent.com/5360060/219932648-2a1b1102-b346-4e8d-ae51-21acb883ca08.png)

**My sponsorship receipt**
![image](https://user-images.githubusercontent.com/5360060/219932619-c0b9a4b8-dbf6-4a42-a304-2c3a72403c36.png)
**Where is it?**
![image](https://user-images.githubusercontent.com/5360060/219932654-86400819-88a1-4dde-ba4a-d912be28f211.png)

**Reply by RakhmatovTemur** [reply]
I have the same problem

**Reply by lexus2k** [reply]
The same problem for me

**Comment by tuves** [comment]
We became aware that Copilot was unavailable in several countries following our efforts to restore access to Ukraine, and on further investigation, we identified the cause as originating from a change we deployed to block Copilot in trade-restricted countries. To prevent this from happening again, we've updated our reporting dashboards to better account for error rates due to trade restrictions and are implementing more robust testing of the trade restriction logic.

**Reply by wuwei-bit** [reply]
china?

**Comment by edakhmetgareev** [comment]
I also got a problem. The trial period was passed and my card was charged 9,68 Eur (I do not why not 10), but I do not see any transactions in the profile and the copilot is not active anymore.

**Reply by hudsonm62** [reply]
I had to contact support, seems the billing system for personal GitHub users is quite brittle

**Reply by lucasriechelmann** [reply]
@edakhmetgareev the price is 10 dollars = 9.68 euros
I am trying to pay but getting errors too. There should be some issues in the billing.

**Reply by lucasriechelmann** [reply]
They might be blocking in some regions

**Reply by lexus2k** [reply]
The same for me, they took my money, but no subscription.  And support stays silent

**Reply by kissmycodee** [reply]
So what is the solution i dont want create credit card for this what options for europe ?

**Comment by maddoxdev** [comment]
The same situation. I chose an annual plan, submitted my billing information, and saw this congratulations page. 
![image](https://github.com/community/community/assets/10694484/7b04e16e-592a-46d8-a7f3-74c97cf45a15)
But the actual payment was not made. They don't want to take my money. And of course the subscription is not active

**Reply by MuhammadEbraheem** [reply]
for me also and iam living in indonesia

**Reply by maddoxdev** [reply]
I solved this problem by using a another country bank card.
Problems was with MasterCard (Turkey). With American MasterCard no problem.

**Comment by akunbeben** [comment]
Does anyone solve this issue? I had the same situation now.

**Reply by lucasriechelmann** [reply]
In the comments you can find a link for the support page I could solve it asking the support.

**Reply by lucasriechelmann** [reply]
Or I find in another topic..I do not remember it

**Comment by lucasriechelmann** [comment]
I contacted the support of GitHub and they solved the problem and I could pay for the monthly plan.

**Reply by kissmycodee** [reply]
What payment method is good if i dont have debit or creditcard ? Also from Europe

**Comment by ahmetcagatay** [comment]
edit: It's just about my virtual card limit. I added some limit and my problem solved.

Same problem... from Turkey. I used virtual card, your cards was normal card or virtual card? Maybe it's about it.

**Comment by chrisk8er** [comment]
I still have the same problem here...

**Comment by miguelvictor** [comment]
I have this problem too ...

**Comment by santiagomalter** [comment]
Same problem :/

**Comment by RakhmatovTemur** [comment]
I have the same problem. The transaction was made and payment went through. I have the receipt showing that payment was done.  I saw the congratulation page too. But I still don't have access to Copilot, and it is asking me to make payment again.

**Reply by njabulobc** [reply]
how did you resolve this

**Comment by ed0001tt** [comment]
i had the same problem. i changed payment method to paypal and it worked.

**Comment by rijalghodi** [comment]
I have same problem

**Comment by RakhmatovTemur** [comment]
I got it fixed. I wrote to the support. They explained it as sync issue in their system.

**Comment by gangjian0917** [comment]
I had the same problem. 
I made a  github copilot individual subscription for one year.  
after two months later， it's unavailable.     the ide shows "No Copilot subsription found"
Can anyone help me?   Thanks

**Reply by lexus2k** [reply]
The same and they are not replying

**Reply by Ali136384** [reply]
same happend to me !

**Comment by Ali136384** [comment]
same problem is here !!!!!!!!!!!!!!!
